### PR TITLE
webui: fix waiting for disabled button in some tests

### PR DESCRIPTION
**Pull request by rvykydal**
Originally the check always passed.

### CODE REVIEW

File: ui/webui/test/check-basic [hunk 1]
```diff
@@ -94,7 +94,7 @@ class TestBasic(anacondalib.VirtInstallMachineCase):
         i.open(step="installation-method")
 
         # Back button should be disabled on the first screen
-        b.wait_visible(f"#installation-back-btn:not([aria-disabled={True}]")
+        b.wait_visible("#installation-back-btn:not([aria-disabled=false]")
```
Comment (KKoukiou) [inline ui/webui/test/check-basic:97]: This does not allign now with the comment in the line above.

File: ui/webui/test/check-storage [hunk 2]
```diff
@@ -109,8 +109,8 @@ class TestStorage(anacondalib.VirtInstallMachineCase, StorageHelpers):
         s.modify_storage()
         # Run the tool
         b.click("#modify-storage-modal-modify-btn")
-        b.wait_visible(f"#modify-storage-modal-modify-btn:not([aria-disabled={True}]")
-        b.wait_visible(f"#storage-modified-modal-rescan-btn")
+        b.wait_visible("#modify-storage-modal-modify-btn:not([aria-disabled=false]")
```
Comment (KKoukiou) [inline ui/webui/test/check-storage:112]: This is pretty useless. Before clicking the button should be always enabled, then get disabled and then re-enabled. So now you 're checking either the pre-disabled state, or the post-disabled state. this is not deterministic, I think we can drop it.

Comment (rvykydal) [reply ui/webui/test/check-storage:112]: I think it is checking the disabled state (I read it disabled is not false)

File: ui/webui/test/check-basic (hunk 1 above)
Comment (KKoukiou) [reply ui/webui/test/check-basic:97]: I was just confused by the double negatio. 

Can you please iron it out a bit:

b.wait_visible(f"#installation-back-btn[aria-disabled=true]")

**Review by KKoukiou — CHANGES_REQUESTED** [review]

**Comment by rvykydal** [comment]
This PR was migrated to the new repository: https://github.com/rhinstaller/anaconda-webui/pull/49

**Pull request closed without merging**